server/routes/record-attendance/index.ts

```diff
@@ -41,6 +47,12 @@ export default function Index({ activitiesService, prisonService }: Services): R
   post('/activities/:id/cancel/confirm', cancelSessionConfirmationRoutes.POST, CancelConfirmForm)
   get('/activities/:id/uncancel', uncancelSessionConfirmationRoutes.GET)
   post('/activities/:id/uncancel', uncancelSessionConfirmationRoutes.POST, UncancelConfirmForm)
+  get('/activities/:id/attendance-details/:attendanceId', attendanceDetailsHandler.GET)
+  post('/activities/:id/attendance-details/:attendanceId', attendanceDetailsHandler.POST)
+  get('/activities/:id/attendance-details/:attendanceId/edit-attendance', editAttendanceHandler.GET)
+  post('/activities/:id/attendance-details/:attendanceId/edit-attendance', editAttendanceHandler.POST, EditAttendance)
+  get('/activities/:id/attendance-details/:attendanceId/remove-pay', removePayHandler.GET)
+  post('/activities/:id/attendance-details/:attendanceId/remove-pay', removePayHandler.POST)
```

Comment:
Shouldn't this request also be validated like edit attendance?

Reply:
Validation added